sdk/servicebus/service-bus/src/receivers/sessionReceiver.ts

```diff
@@ -253,9 +260,15 @@ export class SessionReceiverImpl<ReceivedMessageT extends ReceivedMessage | Rece
    */
   async renewSessionLock(options?: OperationOptions): Promise<Date> {
     this._throwIfReceiverOrConnectionClosed();
-
+    if (!this._messageSession) {
+      const error = new MessagingError(
+        `The session lock has expired on the session with id ${this.sessionId}.`
+      );
+      error.code = "SessionLockLostError";
+      error.retryable = false;
+      throw error;
+    }
```

Comment:
Can we have a test for this? Would probably go to `renewLockSessions.spec.ts` file